src/stories/Library/review/Review.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+export type ReviewProps = {
+  numberOfReviews: number;
+  metaHeadline: string;
+  hearts: 1 | 2 | 3 | 4 | 5 | 6;
+  headlineText: string;
+  bodyText?: string;
+  linkText: string;
+  linkLink?: string;
+};
+
+export const Review: React.FC<ReviewProps> = ({
+  numberOfReviews,
+  metaHeadline,
+  hearts,
+  headlineText,
+  bodyText,
+  linkText,
+  linkLink,
+}) => {
+  const filledHeartsArray = Array.from({ length: hearts }, (_, i) => i + 1);
+  const emptyHeartsArray = Array.from({ length: 6 - hearts }, (_, i) => i + 1);
+  const reviewsArray = Array.from({ length: numberOfReviews }, (_, i) => i + 1);
+  return (
+    <ul className="reviews">
+      {reviewsArray.map((reviewNumber) => {
+        return (
+          <li className="review text-small-caption" key={reviewNumber}>
+            <div className="review__meta mb-8">{metaHeadline}</div>
+            <div className="mb-4">
+              {filledHeartsArray.map((value) => {
+                return (
+                  <img
+                    key={value}
+                    src="icons/basic/icon-heart-filled.svg"
+                    alt="icon-heart-filled"
+                    className="review__heart"
+                  />
+                );
+              })}
+              {emptyHeartsArray.map((value) => {
+                return (
+                  <img
+                    key={value}
+                    src="icons/basic/icon-heart-grey.svg"
+                    alt="icon-heart-grey"
+                    className="review__heart"
+                  />
+                );
+              })}
+            </div>
+            <div className="review__headline mb-8">{headlineText}</div>
```

Comment:
Mmmm I would actually disagree here - we could possibly use h6, but then we would also need to adjust the css to overrule all the predetermined styling that comes with an h6 by default, which we don't want - all the styling we need comes from the parent's class. 
I would even go as far as to say that this is not really a headline. I just called it that because it was the closest thing to a headline I can think of. It's a shorter blob of text above a longer blob of text:
![image](https://user-images.githubusercontent.com/28546954/182195595-c467d00c-532f-47ce-94a0-1a48b0c69a14.png)

But if you still have a different opinion, or have any other concerns in regards to this that I have overseen, I'm listening :) (like I don't know - google will like the site better if we use an h6 or something like that)